test/mock/core/api/transport/api_stub.hpp

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_TEST_MOCK_API_API_STUB_HPP
+#define KAGOME_TEST_MOCK_API_API_STUB_HPP
+
+#include <gmock/gmock.h>
+
+#include <outcome/outcome.hpp>
+
+namespace kagome::api {
+
+  class ApiStub {
```

Comment:
Please leave a short comment with the description of what it is.

Reply:
left